coremltools/converters/mil/mil/ops/defs/activation.py

```diff
@@ -90,22 +91,38 @@ def type_inference(self):
 @register_op(doc_str="")
 class gelu(Operation):
     """
-    Returns the elementwise gaussian error linear unit activation on ``x``.
-
+    Return the elementwise Gaussian error linear unit activation function for ``x``.
+    
+    You can use ``EXACT``, ``TANH_APPROXIMATION``, or ``SIGMOID_APPROXIMATION`` values
+    based on the following formulas:
+    
+    * ``EXACT``:
+    
+    .. math::
+       f(x) = 0.5x\\left ( 1+\\rm{erf}\\left ( \\frac{x}{\\sqrt{2}} \\right ) \\right )
+    
+    * ``TANH_APPROXIMATION``:
+    
+    .. math::
+       f(x) = 0.5x\\left ( 1+\\rm{tanh}\\left ( \\sqrt{2/\\pi}\\left ( x + 0.044715x^3 \\right ) \\right ) \\right )
+    
+    * ``SIGMOID_APPROXIMATION``:
+    
+    .. math::
+       f(x) = x*\\rm{sigmoid}(1.702x)
+
+    
     Parameters
     ----------
     x: tensor<*?, T> (Required)
     mode : const str (Optional)
```

Comment:
This is not getting displayed properly in the preview. somehow the semi-colon is got lost from `modeconst str (Optional)
`

Reply:
That is strange. I'll try a blank line between them. Funny that it should be in this text block but not in others.

Reply:
Found the problem. It doesn't like an extra space before the colon. Now fixed.